graphql/util.go

```diff
@@ -194,7 +194,11 @@ func SchedulePatch(ctx context.Context, patchId string, version *model.Version,
 		}
 	}
 
-	tasks.ExecTasks = model.IncludeDependencies(project, tasks.ExecTasks, p.GetRequester())
+	tasks.ExecTasks, err = model.IncludeDependencies(project, tasks.ExecTasks, p.GetRequester())
```

Comment:
Can you put another comment here saying why we don't return the error, just to be extra careful no one forgets this logic and then adds it?

Reply:
I think it's clear enough from the level that it's logged at. A comment wouldn't add any more info